cli/cmd/testdata/install_helm_crds_output.golden

```diff
@@ -1,4 +1,6 @@
 ---
+# Source: linkerd-crds/templates/gateway.networking.k8s.io/httproute.yaml
+---
```

Comment:
The CRD is not rendering in this test, probably `install_helm_test.go` requires some changes in order to consume the new `values.yaml`